api/src/main/java/org/openmrs/Person.java

```diff
@@ -1055,4 +1055,48 @@ public void setId(Integer id) {
 		
 	}
 	
+	// Deep copy helpers
+	
+	/**
+	 * Creates deep copy of set of addresses.
+	 * 
+	 * @param personAddressesSrc Set of addresses to be copied.
+	 * @return Deep copy of given set of addresses.
+	 */
+	private void deepCopyAddresses(Set<PersonAddress> personAddressesSrc) {
+		for(PersonAddress personAddressSrc : personAddressesSrc) {
```

Comment:
Yes, now it got fixed. Thank you! Shall I push this formatting change? I ask because after your merge my commit will consist only of this additional formatting, it will not indicate my previous changes as I got them now from upstream.